tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -8371,6 +8371,9 @@ declare namespace ts {
     function parenthesizeElementTypeMembers(members: ReadonlyArray<TypeNode>): NodeArray<TypeNode>;
     function parenthesizeTypeParameters(typeParameters: ReadonlyArray<TypeNode> | undefined): MutableNodeArray<TypeNode> | undefined;
     function parenthesizeConciseBody(body: ConciseBody): ConciseBody;
+    function isCommaSequence(node: Expression): node is (BinaryExpression & {
```

Comment:
I don't think this should be included in the declaration file. I'll open a separate issue for that